src/alire/alire-releases.adb

```diff
@@ -459,42 +459,44 @@ package body Alire.Releases is
-            use all type APL.Cardinalities;
-
-            Values : constant Vector :=
-                       R.Labeled_Properties_Vector (No_Properties, -- TODO: what about platform properties?
-                                                    Label);
-         begin
-            --  Sanity checks
-            if APL.Mandatory (Label) and then Values.Is_Empty then
-               raise Program_Error with "Release lacks mandatory label: " & Label'Img;
-            elsif APL.Cardinality (Label) = Unique and then Natural (Values.Length) > 1 then
-               raise Program_Error with "Release property should be unique: " & Label'Img;
-            end if;
-
-            --  Extract either single or vector TOML values
-            if not Values.Is_Empty then
-               declare
-                  TOML_Values : constant TOML.TOML_Value :=
-                                  (case APL.Cardinality (Label) is
-                                      when Unique   => Values.First_Element.To_TOML,
-                                      when Multiple => Values.To_TOML);
-               begin
-                  General.Set (TOML_Keys.Labels (Label),
-                               TOML_Values);
-               end;
-            end if;
-         end;
-      end loop;
-
-      Root.Set ("general",       General);
+      Root.Set (TOML_Keys.General, R.Properties.To_Toml);
+
+--        for Label in Alire.Properties.Labeled.Labels loop
+--           declare
+--              package APL renames Alire.Properties.Labeled;
+--              use all type APL.Cardinalities;
+--
+--              Values : constant Vector :=
+--                         R.Labeled_Properties_Vector (No_Properties, -- TODO: what about platform properties?
+--                                                      Label);
+--           begin
+--              --  Sanity checks
+--              if APL.Mandatory (Label) and then Values.Is_Empty then
+--                 raise Program_Error with "Release lacks mandatory label: " & Label'Img;
+--              elsif APL.Cardinality (Label) = Unique and then Natural (Values.Length) > 1 then
+--                 raise Program_Error with "Release property should be unique: " & Label'Img;
+--              end if;
+--
+--              --  Extract either single or vector TOML values
+--              if not Values.Is_Empty then
+--                 declare
+--                    TOML_Values : constant TOML.TOML_Value :=
+--                                    (case APL.Cardinality (Label) is
+--                                        when Unique   => Values.First_Element.To_TOML,
+--                                        when Multiple => Values.To_TOML);
+--                 begin
+--                    General.Set (TOML_Keys.Labels (Label),
+--                                 TOML_Values);
+--                 end;
+--              end if;
+--           end;
+--        end loop;
+--
+--        Root.Set ("general",       General);
```

Comment:
Have you left this by accident? Commented code looks messy, and no explanation about why it’s there will create doubt in the future when we’ll wonder if we have to leave it here or remove it ^^

Reply:
Right you are, to be deleted